src/api/app/controllers/webui/monitor_controller.rb

```diff
@@ -36,6 +36,7 @@ def index
       @workers_sorted = workers.sort_by { |a| a[0] } if workers
       @available_arch_list = Architecture.available.order(:name).pluck(:name)
     end
+    switch_to_webui2
```

Comment:
We need to add `switch_to_webui2 if Rails.env.development?`, because we are starting this migration.

Reply:
Done! Thanks!